pandas/core/series.py

```diff
@@ -745,12 +745,31 @@ def __array__(self, dtype=None):
             dtype = 'M8[ns]'
         return np.asarray(self.array, dtype)
 
-    def __array_wrap__(self, result, context=None):
+    def __array_wrap__(self, result: np.ndarray, context=None) -> 'Series':
         """
-        Gets called after a ufunc.
+        We are called post ufunc; reconstruct the original object and dtypes.
+
+        Parameters
+        ----------
+        result : np.ndarray
+        context
```

Comment:
I think this will fail validation without indentation, capitalization and punctuation

Reply:
updated with types & doc-string; these are not documented in numpy at all